### ISSUE TITLE
Feature Request: Maxwell log show date as well as time

### ISSUE THREAD

**Issue by jpeyper** [zendesk/maxwell#163]
the current log has this format

```
09:01:07,003 DEBUG SchemaChange - ...
```
it would be nice to have the date included with the time so that you can tell if maxwell is still running correctly and responsively.

**Comment by osheroff** [comment]
@jpeyper, I went to some lengths trying to figure out how to do this with a logj4 `-D` command line flag.  I've returned and can honestly say I'd prefer to burn by eyes out than ever read that manual again.

Now having said that, a workaround is to grab the log4j2.xml from here: https://github.com/zendesk/maxwell/blob/master/src/main/resources/log4j2.xml, copy that into the place where you untarred maxwell, and edit `<PatternLayout pattern="....">` as you see fit.  In future distributions of Maxwell I'll include that file in the root of the tarball for easy editing.  If I can figure out how to not gouge out my eyes with maven.
